packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec

```
@@ -1,12 +1,13 @@
 Name: foreman-obsolete-packages
-Version: 1.6
+Version: 1.7
 Release: 1%{?dist}
 License: MIT
 Summary: A package to obsolete retired packages
 URL: https://github.com/theforeman/foreman-packaging
 BuildArch: noarch
 
 Obsoletes: rubygem-fog-google < 1.19.0-2
+Obsoletes: rubygem-foreman_hooks <= 0.3.17
```

In other places we use the exact release + 1, so:
```suggestion
Obsoletes: rubygem-foreman_hooks < 0.3.17-4
```

This is a common practice to the point that I don't even know how `<=` operates in RPM.